server/sonar-server/src/main/java/org/sonar/server/computation/step/QualityGateMeasuresStep.java

```diff
@@ -170,4 +235,14 @@ public void addEvaluatedCondition(Condition condition, EvaluationResult evaluati
       return evaluatedConditions;
     }
   }
+
+  private enum EvaluatedConditionToCondition implements Function<EvaluatedCondition, Condition> {
+    INSTANCE;
+
+    @Override
+    @Nonnull
+    public Condition apply(@Nonnull EvaluatedCondition input) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this "Nonnull" annotation to honor the overridden method's contract. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2638)